templates/course_homepage/course_homepage.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+  <head>
```

Comment:
You should use a more robust `<head>` here. Type `!` in your Visual Studio Code to try out.